model/internal/otlp_wrapper.go

```diff
@@ -125,7 +126,7 @@ func intHistogramToHistogram(src *otlpmetrics.Metric_IntHistogram) *otlpmetrics.
 	datapoints := []*otlpmetrics.HistogramDataPoint{}
 	for _, datapoint := range src.IntHistogram.DataPoints {
 		datapoints = append(datapoints, &otlpmetrics.HistogramDataPoint{
-			Labels:            datapoint.Labels,
+			Labels:            stringKeyValuePtrArrayToStringKeyValueArray(datapoint.Labels),
```

Comment:
Sorry, I don't understand why does the need to do this arise? Why aren't all StringKeyValue pair lists using either slices of pointers or slices of values? Why do we need to mix and then be forced to do this conversion?

Reply:
This is now fixed.